aspects/aspects.go

```diff
@@ -85,10 +89,52 @@ type Schema interface {
 
 // Directory holds a series of related aspects.
 type Directory struct {
```

Comment:
we need to decide when to do the discussed renaming /directory/bundle/

Reply:
I opened https://github.com/snapcore/snapd/pull/12748. Once it lands in master, I'll merge to this branch so it's up to date